<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMenu</class>
 <widget class="QMainWindow" name="MainMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Araba Kayıt</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Marka</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Model</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Yıl</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>160</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Vites</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>210</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Yakıt Tipi</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineBrand">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineModel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineYear">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>110</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Motor</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineHp">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>30</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>70</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Renk</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>731</width>
      <height>192</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <column>
     <property name="text">
      <string>Marka</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Model</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Yıl</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Vites Tipi</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Yakıt Tipi</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Motor Gücü</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Renk</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btnAdd">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>490</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Ekle</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../free-green-add-button-icon-12023-thumb.png</normaloff>../free-green-add-button-icon-12023-thumb.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDelete">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>490</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Sil</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../delete-icon-close-symbol-vector-no-sign-cancel-symbol-wrong-reject-illustration-delete-icon-close-symbol-vector-no-sign-cancel-131584727.jpg</normaloff>../delete-icon-close-symbol-vector-no-sign-cancel-symbol-wrong-reject-illustration-delete-icon-close-symbol-vector-no-sign-cancel-131584727.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoRepeatDelay">
     <number>398</number>
    </property>
    <property name="autoRepeatInterval">
     <number>200</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btnUpdate">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>490</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Güncelle</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../50787717-update-icon-flat-design-style-eps-10.jpg</normaloff>../50787717-update-icon-flat-design-style-eps-10.jpg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QComboBox" name="comColor">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>70</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Beyaz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Siyah</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Kırmızı</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mavi</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Gri</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comGearType">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>160</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Otomatik</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Manuel</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Yarı Otomatik</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comFuelType">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>210</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Benzin</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Gaz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dizel</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hybrid</string>
     </property>
    </item>
   </widget>
   <widget class="QGroupBox" name="groupFuelCons">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>120</y>
      <width>311</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Yakıt Tüketimi Hesaplama</string>
    </property>
    <widget class="QLineEdit" name="lineKm">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnCal">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Hesapla</string>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="autoRepeat">
      <bool>false</bool>
     </property>
     <property name="autoRepeatDelay">
      <number>398</number>
     </property>
     <property name="autoRepeatInterval">
      <number>200</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineResult">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>100</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Kilometre</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Tüketim</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>822</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
